apps/frontend/src/components/ViewFormsList/index.tsx

```diff
@@ -24,13 +36,13 @@ export default function ViewFormsList () {
         </Thead>
         <Tbody>
           {
-            testData.map(child =>
-              <Tr key={child.id}>
-                <Td>{new Date(child.date).toLocaleDateString()}</Td>
-                <Td>{child.childsName}</Td>
-                <Td>{new Date(child.dob).toLocaleDateString()}</Td>
-                <Td>{child.hospital}</Td>
-                <Td>{`${child.address.city}, ${child.address.state}`}</Td>
+            forms.map(form =>
```

Comment:
could you add something like this so that the page displays a loading spinner while forms are being fetched from the backend?  it seems like right now there's a little bit of lag and that could potentially get worse with many more forms 

https://github.com/Code-4-Community/constellation/pull/39/files#diff-4ee101d7d403a1c17257fabf1f4a0cbe4358782c3c0a2f7d1041de8030cb4136R45-R54